MvvmCross.iOS.Support/MvvmCross.iOS.Support.csproj

```diff
@@ -83,6 +83,7 @@
     <Compile Include="SidePanels\MvxPanelPopToRootPresentationHint.cs" />
     <Compile Include="SidePanels\MvxPanelPresentationHint.cs" />
     <Compile Include="SidePanels\IMvxSideMenu.cs" />
+    <Compile Include="Views\MvxUIRefreshControl.cs" />
```

Comment:
Shouldn't this be in the main repo? Isn't UIRefreshControl widely available in iOS?